BraintreeCore/src/main/java/com/braintreepayments/api/core/BraintreeClient.kt

```diff
@@ -63,24 +67,31 @@ open class BraintreeClient @VisibleForTesting internal constructor(
         analyticsClient = params.analyticsClient,
         httpClient = params.httpClient,
         graphQLClient = params.graphQLClient,
-        browserSwitchClient = params.browserSwitchClient,
         configurationLoader = params.configurationLoader,
         manifestValidator = params.manifestValidator,
         returnUrlScheme = params.returnUrlScheme,
-        braintreeDeepLinkReturnUrlScheme = params.braintreeReturnUrlScheme
+        braintreeDeepLinkReturnUrlScheme = params.braintreeReturnUrlScheme,
+        appLinkReturnUri = params.appLinkReturnUri
     )
 
     /**
      * @suppress
      */
     @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
-    constructor(context: Context, authorization: String) :
-        this(
-            BraintreeOptions(
-                context = context,
-                authorization = Authorization.fromString(authorization)
-            )
+    @JvmOverloads
+    constructor (
+        context: Context,
+        authorization: String,
+        returnUrlScheme: String? = null,
+        appLinkReturnUri: Uri? = null,
```

Comment:
Do we want to make these required in the constructor vs optional?

Reply:
In `v5` the `appLinkUrl` is required in the `PayPalClient` constructor - the merchant won't instantiate a `BraintreeClient` anymore it's all internal so optional because the other feature clients don't support app links yet

Reply:
Gotcha - sorry didn't realize this was an internal constructor 🙈